faqs/galaxy/histories_transfer_entire_histories_from_one_galaxy_instance_to_another.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Transfer entire histories from one Galaxy instance to another
```

Comment:
I think the word "instance" is a bit on the technical side for most users, you can just leave out the word? or use 'server' instead

```suggestion
title: Transfer entire histories from one Galaxy to another
```